_site/study-groups/multivariate-blind-subspace.html

```diff
@@ -0,0 +1,1161 @@
+<!DOCTYPE html>
```

Comment:
So this file should not be edited either. The contents of _site are auto-generated by jekyll during compilation. Instead, please edit this file- `_study-groups/multivariate-blind-subspace.md`. I did not notice it earlier, sorry!

Reply:
How should I edit the `.md` file?

Reply:
You can use markdown formatting syntax to edit the file. Here's a handy cheat sheet- https://enterprise.github.com/downloads/en/markdown-cheatsheet.pdf , you can google for more ;)

Reply:
Yes, I think you can copy-paste another person's biography and then edit with your info.

Reply:
But I'm still not sure what to do. There are no other biographies in the `.md` file.

Reply:
No worries, check this file- `_data/subspace_volunteers.yml`. You should add your block here, then it will show up on the appropriate page.

Reply:
Done. Ty.